AssociationCollection.php

```diff
@@ -327,4 +327,20 @@ public function getIterator()
     {
         return new ArrayIterator($this->_items);
     }
+
+    /**
+     * Get the _name of all the associations in the collection.
+     *
+     * @return array
+     */
+    public function getAssociationsName()
```

Comment:
Naming this method `getNames()` would be better. So usage would be `$model->associations()->getNames()`.